hd/lang/lex_utf8.txt

```diff
@@ -28,6 +28,10 @@ fr: dictionnaire des %s
 en: compiled on
 fr: compilé le
 
+    commited on
```

Comment:
The href links works only for commits available in geneweb/geneweb, but I guess this is ok.